hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/CountryUserAccountCreationFailedState.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.ida.hub.policy.domain.state;
+
+import com.google.common.base.Optional;
+import org.joda.time.DateTime;
+import uk.gov.ida.hub.policy.domain.SessionId;
+
+import java.net.URI;
+
+public class CountryUserAccountCreationFailedState extends AbstractUserAccountCreationFailedState implements EidasUnsuccessfulJourneyState {
+
+    private static final long serialVersionUID = -2561859918430555052L;
+
+    public CountryUserAccountCreationFailedState(
+        String requestId,
+        String authnRequestIssuerEntityId,
+        DateTime sessionExpiryTimestamp,
+        URI assertionConsumerServiceUri,
+        Optional<String> relayState,
+        SessionId sessionId) {
+
+        super(requestId, authnRequestIssuerEntityId, sessionExpiryTimestamp, assertionConsumerServiceUri, relayState, sessionId, true);
```

Comment:
I don't think it's a safe assumption to just set `true` for the `transactionSupportsEidas` argument.

Reply:
Hmm.... maybe it is....

Reply:
If you get to this stage and the transaction doesn't support eIDAS then you have a serious problem! All the eIDAS states shouldn't even have this flag. Of course the transaction has to support eIDAS to even be in these states. Once you've shown the country picker, we know this is always true